board/views.py

```diff
@@ -1,25 +1,25 @@
 import datetime
+import gamestore.settings as settings
 from django.shortcuts import render
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
-from gamestore.settings import BEARER_TOKEN, API_CLIENT_ID, API_SECRET_KEY
-from .apiwrappers import TwitterWrapper, IgdbWrapper
+from .api import igdbapi, twitterapi
 
 
-twitter_wrapper = TwitterWrapper(BEARER_TOKEN)
-igdb_wrapper = IgdbWrapper(API_CLIENT_ID, API_SECRET_KEY)
+twitter_wrapper = twitterapi.TwitterWrapper(settings.API_TWITTER_TOKEN)
+igdb_wrapper = igdbapi.IgdbWrapper(settings.API_IGDB_CLIENT_ID, settings.API_IGDB_TOKEN)
 
 
 class Game:
     def __init__(self, game_id):
-        res = igdb_wrapper.get_games_by_id(game_id)[0]
+        res = igdb_wrapper.get_game_by_id(game_id)[0]
```

Comment:
I think this method should return either single game or `None`, no need to return list by id, right?